src/templates/bootstrap/components/well.html

```diff
@@ -0,0 +1,7 @@
+<div [formGroup]="form" class="form-group">
+    <div *ngFor="let control of component.control.controls;" class="well">
```

Comment:
You should really only need the ```<formio-components>``` directive here.  Refer to Refer to https://github.com/formio/ng2-formio/blob/master/src/templates/bootstrap/components/columns.html